source/_layouts/default.html

```diff
@@ -55,7 +55,7 @@
 
     <div id="doc-switchers" class="container">
       <div class="col-md-8">{% include template/doc_switcher.html %} {% include template/version_switcher.html %}</div>
-      <div class="col-md-4"></div>
+      <div class="col-md-4"><div class="visible-sm-inline visible-xs-inline"><a href="#document-nav">Jump to table of contents</a></div></div>
```

Comment:
What about "product navigation", "documentation outline", or even just "other pages"? Something more descriptive of what it contains, as opposed to where it might be located in the layout.